docs/dag-run.rst

```diff
@@ -0,0 +1,191 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+DAG Runs
+=========
+A DAG Run is an object representing an instantiation of the DAG in time.
+
+Each DAG may or may not have a schedule, which informs how DAG Runs are
+created. ``schedule_interval`` is defined as a DAG argument, and receives
+preferably a
+`cron expression <https://en.wikipedia.org/wiki/Cron#CRON_expression>`_ as
```

Comment:
I know it might be biased to refer to certain projects, but it might be nice to reference [crontab.guru](https://crontab.guru/) as it helps people create crons and displays the period at which they would run.

Reply:
Great idea, but probably best to link to crontab.guru in the wikipedia page itself.

Reply:
😢

Reply:
I don't see a problem linking to crontab.guru as well in an aside.